Tiltfile

```diff
@@ -170,7 +168,6 @@ cmd_button(
     kubectl rollout restart deployment temporal; \
     kubectl rollout restart deployment enduro; \
     kubectl rollout restart statefulset enduro-{pres_sys}; \
-    kubectl rollout restart deployment dex; \
```

Comment:
We did it for Dex because we were recreating its MySQL DB, something we probably didn't need. For Keycloack, I can only see the need to restart it if you make changes to the imported realm.